ocs_ci/templates/workloads/Files_create_delete.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
```

Comment:
this kind of create,delete seems to be only needed for this specific bz. please move this code inside the test function